src/components/app-header/composables/useRouteDependentSettings.ts

```diff
@@ -0,0 +1,19 @@
+import { computed, Ref } from 'vue'
+import { RouteLocationNormalizedLoaded, useRoute } from 'vue-router'
+import { APP_NAVIGATION_COLOR_TYPES } from '../../shared/app-navigation/AppNavigation.types'
+import { ROUTE_NAMES } from '../../../router/router.constants'
```

Comment:
I could add global path for import @shared and @router. Still have an error "Cannot find module '@router/router.constants.ts' or its corresponding type declarations." So I decided to use simple import everywere.